Code/Editor/Plugins/EditorAssetImporter/AssetImporterPlugin.cpp

```diff
@@ -102,21 +99,53 @@ AZStd::unique_ptr<AZ::DynamicModuleHandle> AssetImporterPlugin::LoadSceneLibrary
 
 QMainWindow* AssetImporterPlugin::EditImportSettings(const AZStd::string& sourceFilePath)
 {
-    const QtViewPane* assetImporterPane = GetIEditor()->OpenView(m_toolName.c_str());
-
-    if(!assetImporterPane)
+    if (!m_assetImporterWindow)
     {
         return nullptr;
     }
 
-    AssetImporterWindow* assetImporterWindow = qobject_cast<AssetImporterWindow*>(assetImporterPane->Widget());
+    AssetImporterWindow* assetImporterWindow = qobject_cast<AssetImporterWindow*>(m_assetImporterWindow);
     if (!assetImporterWindow)
     {
         return nullptr;
     }
 
     assetImporterWindow->OpenFile(sourceFilePath);
-    return assetImporterWindow;
+    return m_assetImporterWindow;
+}
+
+QMainWindow* AssetImporterPlugin::OpenImportSettings()
+{
+    // Can only be one asset importer window at a time
+    if (m_assetImporterWindow)
+    {
+        return nullptr;
+    }
+
+    m_assetImporterWindow = new AssetImporterWindow();
+    return m_assetImporterWindow;
```

Comment:
What are the implications of allocating the asset import window from the beginning and showing or hiding it instead of deleting and reconstructing?